src/modules/database/entities/question.entity.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  Column, Entity, ManyToOne, PrimaryGeneratedColumn,
+} from 'typeorm';
+import { Category } from './category.entity';
+import { GenderEnum } from '../../../shared/enums/gender.enum';
+import { User } from './user.entity';
+import { Answer } from './answer.entity';
+
+@Entity({ name: 'question' })
+export class Question {
+  @PrimaryGeneratedColumn('uuid')
+  @ManyToOne(() => Answer, (answer) => answer.id)
```

Comment:
OneToMany for start, then you need to past it like 
@OneToMany(() => Answer, (answer) => answer.id)
  answers: Answer[];